### PR TITLE
v.out.ogr: adds basic unit tests

### PR DESCRIPTION
**Pull request by ldesousa**
Test export and import of GeoPackage format.

### CODE REVIEW

File: vector/v.out.ogr/testsuite/test_v_out_ogr.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        )
+        for p in Path(".").glob("%s*" % self.test_map):
+            p.unlink()
+
+    def test_1(self):
+        self.assertModule(
+            "v.out.ogr",
+            "Export to GeoPackage Format",
+            input=self.test_map,
+            output="%s.gpkg" % self.test_map,
```
Comment (wenzeslaus) [inline vector/v.out.ogr/testsuite/test_v_out_ogr.py:62]:
```suggestion
            output=f"{self.test_map}.gpkg"
```

Same also elsewhere.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
Just a detail. Pylint would report the same.

**Review by wenzeslaus — COMMENTED** [review]
Please, see the other PR for comments on comments.

File: vector/v.out.ogr/testsuite/test_v_out_ogr.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    @classmethod
+    def tearDownClass(cls):
+        """!Remove the temporary region"""
+        cls.del_temp_region()
+
+    def tearDown(self):
+        self.runModule(
+            "g.remove", type="vector", flags="f", pattern=f"{self.temp_import}*"
+        )
+        for p in Path(".").glob(f"{self.test_map}*"):
```
Comment (echoix) [inline vector/v.out.ogr/testsuite/test_v_out_ogr.py:54]:
```suggestion
        for p in Path().glob(f"{self.test_map}*"):
```

**Review by echoix — COMMENTED** [review]
A ruff fix suggestion

**Comment by echoix** [comment]
I'm about to approve it, but what were the comments of the other PR that Vaclav was referring to? They have been addressed?

**Review by echoix — DISMISSED** [review]

File: vector/v.out.ogr/testsuite/test_v_out_ogr.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Use temporary region settings"""
+        cls.use_temp_region()
+
+    @classmethod
+    def tearDownClass(cls):
+        """!Remove the temporary region"""
```
Comment (wenzeslaus) [inline vector/v.out.ogr/testsuite/test_v_out_ogr.py:47]:
```suggestion
        """Remove the temporary region"""
```

**Review by wenzeslaus — APPROVED** [review]
This looks good and the comments look good. Thank you for the updates!

**Pull request merged**